about.html

```diff
@@ -65,6 +65,33 @@ <h2>Where are we now?</h2>
         <div class="about-faqs row pt-2">
           <h2>FAQs</h2>
           <div class="accordion" id="accordionExample">
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
```

Comment:
The values of `data-bs-target`, `aria-controls` on this line, as well as `id` on line 74 need to all be the same unique string. That's how the bootstrap library knows what clicks correspond to what expands/collapses. `collapseTwo` I think was already used. You can call it anything, e.g. `guillaume-rulez-1`